examples/apps/cchmc_ped_abd_ct_seg_app/scripts/map_build.sh

```
@@ -12,18 +12,28 @@
 # assign command-line arguments to variables
 tag_prefix=$1
 image_version=$2
-sdk_version=$3
+monai_deploy_app_sdk_version=$3
+cuda_version=$4
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 # load in environment variables
-source .env
+source "${script_dir}/../.env"
```

Shellcheck (0.11.0)

[info] Not following: ./../.env was not specified as input (see shellcheck -x).

(SC1091)